packages/dds/tree/src/feature-libraries/flex-tree/lazyEntity.ts

```diff
@@ -94,14 +91,13 @@ export abstract class LazyEntity<TSchema = unknown, TAnchor = unknown>
 	}
 
 	protected abstract [tryMoveCursorToAnchorSymbol](
-		anchor: TAnchor,
 		cursor: ITreeSubscriptionCursor,
 	): TreeNavigationResult;
 
 	/**
 	 * Called when disposing of this target, iff it has an anchor.
 	 */
-	protected abstract [forgetAnchorSymbol](anchor: TAnchor): void;
+	protected abstract [forgetAnchorSymbol](): void;
```

Comment:
Why was this change made? I see that LazyTreeNode (and thus all nodes in the tree) gained another field, which would be nice to avoid for memory pressure.

Reply:
I posed the same question to @CraigMacomber [here](https://github.com/microsoft/FluidFramework/pull/20286#discussion_r1536203885). Brought this straight from his proposal and I'm also not sure about the reasoning.